README.md

```diff
@@ -89,7 +89,7 @@ ort-scan:
     VCS_URL: 'ssh://git@github.com:jshttp/mime-types.git'
     VCS_REVISION: 'ef932231c20e716ec27ea159c082322c3c485b66'
     ALLOW_DYNAMIC_VERSIONS: 'true'
-    ORT_CLI_ANALYZE_ARGS: '-P ort.analyzer.enabledPackageManagers=NPM,Yarn,Yarn2'
+    ORT_CLI_ARGS: '-P ort.analyzer.enabledPackageManagers=NPM,Yarn,Yarn2 -P ort.forceOverwrite=true --stacktrace'
```

Comment:
Why did you add `-P ort.forceOverwrite=true --stacktrace` ?

Reply:
I do this to retain the default value for ORT_CLI_ARGS see https://github.com/oss-review-toolkit/ort-ci-gitlab/blob/main/templates/ort-scan.yml#L82

Reply:
It'd be nice to either mention in the commit message or to make it a separate commit IMHO.